src/shared/types.ts

```diff
@@ -17,15 +17,15 @@ export interface Article {
 }
 
 export interface ArticleContent {
-  content_raw: string
+  content_raw: string;
 }
 
 export interface Issue {
-  id: number;
-  publish_date: string;
+  id?: number;
+  publish_date?: string;
   volume_num: number;
   issue_num: number;
```

Comment:
issue_num should be a string (e.g. for election issues, we would publish `v111iE`).